react/features/prejoin/utils.js

```diff
@@ -0,0 +1,802 @@
+        code: 'sj' },
+    { name: 'Swaziland',
+        dialCode: '268',
+        code: 'sz' },
+    { name: 'Sweden',
+        dialCode: '46',
+        code: 'se' },
+    { name: 'Switzerland',
+        dialCode: '41',
+        code: 'ch' },
+    { name: 'Syrian Arab Republic',
+        dialCode: '963',
+        code: 'sy' },
+    { name: 'Taiwan',
+        dialCode: '886',
+        code: 'tw' },
+    { name: 'Tajikistan',
+        dialCode: '992',
+        code: 'tj' },
+    {
+        name: 'Tanzania, United Republic of Tanzania',
+        dialCode: '255',
+        code: 'tz'
+    },
+    { name: 'Thailand',
+        dialCode: '66',
+        code: 'th' },
+    { name: 'Timor-Leste',
+        dialCode: '670',
+        code: 'tl' },
+    { name: 'Togo',
+        dialCode: '228',
+        code: 'tg' },
+    { name: 'Tokelau',
+        dialCode: '690',
+        code: 'tk' },
+    { name: 'Tonga',
+        dialCode: '676',
+        code: 'to' },
+    { name: 'Trinidad and Tobago',
+        dialCode: '1868',
+        code: 'tt' },
+    { name: 'Tunisia',
+        dialCode: '216',
+        code: 'tn' },
+    { name: 'Turkey',
+        dialCode: '90',
+        code: 'tr' },
+    { name: 'Turkmenistan',
+        dialCode: '993',
+        code: 'tm' },
+    { name: 'Turks and Caicos Islands',
+        dialCode: '1649',
+        code: 'tc' },
+    { name: 'Tuvalu',
+        dialCode: '688',
+        code: 'tv' },
+    { name: 'Uganda',
+        dialCode: '256',
+        code: 'ug' },
+    { name: 'Ukraine',
+        dialCode: '380',
+        code: 'ua' },
+    { name: 'United Arab Emirates',
+        dialCode: '971',
+        code: 'ae' },
+    { name: 'United Kingdom',
+        dialCode: '44',
+        code: 'gb' },
+    { name: 'United States',
+        dialCode: '1',
+        code: 'us' },
+    { name: 'Uruguay',
+        dialCode: '598',
+        code: 'uy' },
+    { name: 'Uzbekistan',
+        dialCode: '998',
+        code: 'uz' },
+    { name: 'Vanuatu',
+        dialCode: '678',
+        code: 'vu' },
+    {
+        name: 'Venezuela, Bolivarian Republic of Venezuela',
+        dialCode: '58',
+        code: 've'
+    },
+    { name: 'Vietnam',
+        dialCode: '84',
+        code: 'vn' },
+    { name: 'Virgin Islands, British',
+        dialCode: '1284',
+        code: 'vg' },
+    { name: 'Virgin Islands, U.S.',
+        dialCode: '1340',
+        code: 'vi' },
+    { name: 'Wallis and Futuna',
+        dialCode: '681',
+        code: 'wf' },
+    { name: 'Yemen',
+        dialCode: '967',
+        code: 'ye' },
+    { name: 'Zambia',
+        dialCode: '260',
+        code: 'zm' },
+    { name: 'Zimbabwe',
+        dialCode: '263',
+        code: 'zw' }
+];
+
+const countriesByCodeMap = countries.reduce((result, country) => {
+    result[country.dialCode] = country;
+
+    return result;
+}, {});
+
+/**
+ * Creates a map between country dial codes and country objects.
+ *
+ * @returns {Object}
+ */
+function getCodesByNumbersMap() {
```

Comment:
if you constructed the `countriesByCodeMap` to be static (which was a good idea, I think), why didn't you make this a const map too?

Reply:
Yes indeed. Thanks for the suggestion. I changed it.